app.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const Mailgun = require('mailgun-js')
+
+const app = express();
+
+const api_key = '317a0f64411bd75d7598da6be916803b-46ac6b00-50f5a2ee';
```

Comment:
You will need to import from .env file, which is written like this require('dotenv').config(). Inside of .env make sure you create API_KEY= '317....'

Reply:
`process.env.MAIL_KEY`

Reply:
`./lib/mailer.js`